en/docs/develop/customizing-authentication-error-messages.md

```diff
@@ -19,14 +19,15 @@ those errors. There are three types of custom errors handled here:
 
 Do the following to customize these error messages.
 
-Enable the following parameter in the
-`         <IS_HOME>/repository/conf/identity/application-authentication.xml        `
-file if you need customized error messages.
-
-``` xml
-<AuthenticatorConfig name="BasicAuthenticator" enabled="true"> 
-    <Parameter name="showAuthFailureReason">true</Parameter>
-</AuthenticatorConfig>
+Add the following properties to the `deployment.toml` file found in the `<IS_HOME>/repository/conf` folder and enable the authenticator to be able to customize error messages.
+
+``` toml
+[authentication.authenticator.basic] 
+name ="BasicAuthenticator"
+enable=true
+[authentication.authenticator.basic.parameters]
+auth_mechanism= "basic"
+UserNameAttributeClaimUri = ""
 ```
```

Comment:
[authentication.authenticator.basic.parameters]
showAuthFailureReason = true

this should be ebough. verify before adding

Reply:
fixed with 0b4a567974f56d3b0c866d3f7286017c6ff55bc2